CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Filter columns, e.g. `?select=col1,col2` - @ruslantalpa
 - Does not execute the count total if header "Prefer: count=none" - @diogob
 
+### Removed
+- Remove versioning feature - @calebmer
+
+### Fixed
+- Tolerate a missing role in user creation - @calebmer
```

Comment:
This is the cause of the merge conflict, because this fixed entry is already in the v3 branch and somehow this commit says to add it again.